src/datadoc/backend/statistic_subject_mapping.py

```diff
@@ -39,14 +40,16 @@ def __init__(self, source_url: str) -> None:
         self.statistic_short_name_secondary_subject_mapping: dict[str, str] = {
```

Comment:
We should remove lines 39-42, they're left over from my original experimentation.

Reply:
statistic_short_name_secondary_subject_mapping is used in the get primary subjects, but it seems that it is never updated, is this meant to be done like this or is this also code from the original experimentation?